images/macos/software-report/SoftwareReport.Android.psm1

```diff
@@ -172,11 +172,11 @@ function Get-AndroidNDKVersions {
     if ($os.IsLessThanBigSur) {
         # Hardcode NDK 15 as a separate case since it is installed manually without sdk-manager (to none default location)
         $versions += "15.2.4203891"
+    }
 
-        $ndkFolderPath = Join-Path (Get-AndroidSDKRoot) "ndk"
-        Get-ChildItem -Path $ndkFolderPath | ForEach-Object {
-            $versions += $_.Name
-        }
+    $ndkFolderPath = Join-Path (Get-AndroidSDKRoot) "ndk"
+    Get-ChildItem -Path $ndkFolderPath | ForEach-Object {
```

Comment:
I think we should remove `ndk-bundle` in 182 line. Since `ndk-bundle` is symlink for something in ndk folder, it will produce duplicate

Reply:
Strangely _it doesn't_ 
![image](https://user-images.githubusercontent.com/48208649/106448975-8721aa80-6494-11eb-8d27-ae9b6bb183d0.png)
I'll take a look more carefully

Reply:
Oh, I've got it — as we don't install `ndk-bundle` package anymore `$packageInfo` doesn't contain `ndk-bundle` entry.
I removed the section to avoid such confusion

Reply:
Thanks!